tests/integration_tests/functional/test_initrd.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+"""Tests for initd."""
+
+import pytest
+from host_tools.network import SSHConnection
+from framework.microvm import Serial
+
+INITRD_FILESYSTEM = "rootfs"
+
+
+@pytest.mark.xfail(raises=ConnectionError)
```

Comment:
Why do you expect this test to fail?

Reply:
I have the SSH connection to the guest fail depending on the kernel / machine used to test. 
What I could see it's that it takes a while for the SSH server to come up, and I suspect it may be because the random source is not generating fast enough. There was a discussion about that on the Slack channel about 1 month ago.

Reply:
Can we just increase the timeout instead? It might be that the tests never get to run which I don't think it's the best solution.

Reply:
Let me do some more tests and I'll tell you